### DISCUSSION TITLE
Quick Xray question

### DISCUSSION THREAD

**Discussion by Gamemassa** [IrisShaders/Iris#2055] [Q&A]
Hi, quick (hopefully) question: I know Oculus is an _unofficial_ Forge port of Iris (and I posted this in their GitHub and they're looking into it) but I guess I was just hoping you guys might have some general insight as well, since theoretically both mods are similar. 

I have Iris in my custom Fabric modpack, and I also use ATE48's Xray mod (and I posted all this in his GitHub too), which is designed to hide/show blocks based on a list in its config file, and it works perfectly with Iris (even the Fullbright, Cave-viewer and other functions all work too), but I also have a custom Forge modpack that uses Oculus and in that modpack none of ATE48's Xray mod functions work (except I think the ESP function because that just outlines enemies).

Like I said, I was really just hoping you guys might have a general idea as to what might be causing the Xray mod to not work (and if it's not something that can be fixed, it's not a big deal). And/or if the same bug had happened with Iris in the past, maybe whatever you guys did to fix it, something similar could be done by the Oculus team on their mod.

Also for context, both my Forge and Fabric modpacks use MC 1.20.1, if that helps.

**Comment by douira** [comment]
You're probably off best asking the maintainer of Oculus about this, as in Forge many things are different and Oculus also has differences in implementation.

**Comment by Gamemassa** [comment]
Gotcha. Well thanks anyway for the quick reply!

**Comment by IMS212** [comment]
Iris does not support cheats.

**Reply by Gamemassa** [reply]
You don't need to be rude. I only use Xray in my own singleplayer worlds for exploration and debugging purposes (i.e. to make sure certain modded structures are generating correctly). Is that really so bad?

**Reply by douira** [reply]
Nothing about what IMS said was rude, it's just a matter-of-fact statement.
